rotkehlchen/tests/api/test_location_asset_mappings.py

```diff
@@ -12,7 +12,7 @@
 if TYPE_CHECKING:
     from rotkehlchen.api.server import APIServer
 
-NUM_ASSETS_MAPPINGS_V1_37: Final = 2023
+NUM_ASSETS_MAPPINGS_V1_37: Final = 2024
```

Comment:
Hmmm this variable name is now wrong. Is it supposed to be `NUM_SHIPPED_ASSETS_MAPPING`, so the number of the included global.db? Or the one in 1.37?